### DISCUSSION TITLE
symbol-dapp control panel

### DISCUSSION THREAD

**Discussion by aleixmorgadas** [symbol-dapp/community#7] [Framework Ideas]
Even though any Symbol Dapp is free to use their own UX/UI, I think a common control panel UI would be valuable.

I'm thinking about a module to behave as a plug and play page for Vue or React that adds some features to a known URL like `https://symbol.directory/dapp-control-panel`. There, people will find useful features like:

- Debugging tools
- State Control
- Cryptographic proofs
- etc

What do you think?

**Comment by evias** [comment]
Interesting indeed, some backend information is definitely useful.

- Do you make this a *governed* and *protected* sub-app ?
- Or are you thinking about a *Statistics* backend that is accessible publicly too?

+++ on this discussion with important items like **State Control** and **Cryptographic proofs** ; items for which there is few resources available at the time of writing.
